src/lucky/context_extensions.cr

```diff
@@ -1,24 +1,27 @@
-require "./session/**"
-
 class HTTP::Server::Context
   DEBUG_COLOR = :green
-  setter session : Lucky::Session::AbstractStore?
-  setter cookies : Lucky::Cookies::Store?
-  setter flash : Lucky::Flash::Store?
+  setter cookies : Lucky::CookieJar?
+  setter session : Lucky::SessionCookie?
+  setter flash : Lucky::FlashStore?
 
   getter debug_messages : Array(String) = [] of String
   property? hide_from_logs : Bool = false
 
   def cookies
-    @cookies ||= Lucky::Cookies::Store.build(request, Lucky::Server.settings.secret_key_base)
+    @cookies ||= Lucky::Cookies::Processors::Encryptor.read(
```

Comment:
I'd suggest using `abstract class Lucky::Cookies::Processor` along with composition pattern, which will decouple processor from the `CookieJar` class itself.